### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by vijithassar**
lmao

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,48 +2,104 @@
 
 Self-documenting Node scripts through literate programming. Based on [lit](https://github.com/vijithassar/lit) by [Vijith Assar](https://twitter.com/vijithassar).
 
+## Overview
 
-## Quickstart
+`lit-node` is a lightweight wrapper for [Node.js](https://nodejs.org/en/) which allows you to import code blocks from [Markdown](https://daringfireball.net/projects/markdown/syntax) documents using the `require()` function. This enables first-class support for simple [literate programming](https://en.wikipedia.org/wiki/Literate_programming), a software development technique which emphasizes clear written documentation. It's sort of like [Jupyter](http://jupyter.org/) notebooks for Node. Jupyter... nodebooks? *[todo: should we delete this pun? it's really stupid.]*
```
Comment (Rich-Harris) [inline README.md:7]: should we mention jupyter at all? not sure the comparison is warranted — jupyter is a *lot* more sophisticated than this 20 line hack

**Review by Rich-Harris — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (vijithassar) [reply README.md:7]: I was just thinking that putting a rambling pun in the intro sort of demonstrates literate programming, no problem if you're not on board with the comparison and would rather remove it.

Comment (Rich-Harris) [reply README.md:7]: I like the idea of leaving the 'todo' there

**Pull request merged**